libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -201,6 +225,30 @@ void WaveTrack::Interval::TrimRightTo(double t)
       GetClip(channel)->TrimRightTo(t);
 }
 
+void WaveTrack::Interval::SetTrimLeft(double t)
+{
+   for(unsigned channel = 0; channel < NChannels(); ++channel)
+      GetClip(channel)->SetTrimLeft(t);
```

Comment:
`ForEachClip` might be used more consistently in these new functions, but that's not important.

Also I would redefine `ForEachClip` as an inline function, deducing the type of the argument as a template parameter, avoiding the construction of a `std::function` object.  But this is not important.